tests/Unit/inc/Engine/Optimization/Minify/CSS/Minify/optimize.php

```diff
@@ -55,5 +55,10 @@ public function testShouldMinifyCSS( $original, $minified, $cdn_host, $cdn_url,
 			$minified,
 			$this->minify->optimize( $original )
 		);
+
+		$this->assertTrue( $this->filesystem->exists( 'wordpress/wp-content/cache/min/1/wp-content/themes/twentytwenty/style-dcd1a95e5d432b5d300d7c2f216d7150.css' ) );
```

Comment:
Move the files in fixtures and use a loop to parse and make the assertTrue. What do you think?